variables.tf

```diff
@@ -0,0 +1,65 @@
+variable "deletion_window_in_days" {
+  type        = number
+  default     = 10
+  description = "Duration in days after which the key is deleted after destruction of the resource"
+}
+
+variable "enable_key_rotation" {
+  type        = bool
+  default     = true
+  description = "Specifies whether key rotation is enabled"
+}
+
+variable "description" {
+  type        = string
+  default     = "Parameter Store KMS master key"
```

Comment:
Please correct it, its not `Parameter store`

Reply:
Done